2800_app.js

```diff
@@ -7,7 +7,7 @@ const { JSDOM } = require('jsdom');
 const { connected } = require("process");
 
 const sqlAuthentication = { // sql connection settings
-    host: "localhost",
+    host: "127.0.0.1",// for Mac os, Taylor changes this. 

```

Comment:
This should be left as localhost for now. You can keep those changes locally though